presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,28 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
```

Comment:
You could put those in a separate utility class and make it support more functions so that we could standardize it.